### PR TITLE
=doc #280,#397 extend migration guide

### PR DESCRIPTION
**Pull request by jrudolph**
Add some overview documentation of the changes and, in particular, point
to further documentation about entity streaming.

### CODE REVIEW

File: docs/src/main/paradox/scala/http/migration-guide/migration-from-spray.md [hunk 1]
```diff
@@ -202,3 +240,30 @@ toStrict {
...
+IO(Http)(system) ! Http.Bind(service, "0.0.0.0", port = 8080)
+```
+
+needs to be changed into:
+
+```scala
+Http().bindAndHandle(routes, "0.0.0.0", port = 8080)
+```
+
+### Other removed features
```
Comment (2beaucoup) [inline docs/src/main/paradox/scala/http/migration-guide/migration-from-spray.md]: Should we mention proxy support here?

**Comment by ktoso** [comment]
Refs #280 #397

**Comment by ktoso** [comment]
LGTM, nice! I added a note on the proxy support as @2beaucoup mentioned

**Comment by jrudolph** [comment]
:+1:

**Pull request merged**